index.html

```diff
@@ -1430,9 +1441,48 @@
 
 ## Harassment
 
-<aside class="issue">
-  This section has not been written.
-</aside>
+Online <dfn>harassment</dfn> is the "pervasive or severe targeting of an individual or group online
+through harmful behavior" [[PEN-Harassment]]. Harassment is a prevalent problem on the Web,
+particularly via social media. Harassment may be more severe and its consequences more impactful for
+LGBTQ people and women [[?PEW-Harassment]].
+
+<a>Harassment</a> is both a violation of privacy itself and can be a consequence of other violations
```

Comment:
```suggestion
<a>Harassment</a> is both a violation of privacy itself and can be magnified or facilitated by other violations
```

I don't think that it's really a consequence of other privacy violations but they can contribute.